system-addon/content-src/components/TopSites/_TopSites.scss

```diff
@@ -326,13 +321,54 @@
 }
 
 .topsite-form {
+  .form-input-container {
+    display: flex;
+    max-width: 492px;
+    margin: 0 auto;
+    padding: 32px;
+
+    .top-site-outer {
+      width: 96px;
+      padding: 0;
+      margin: 64px 0 0;
+      margin-inline-start: 32px;
+      list-style: none;
+
+      a {
+        cursor: default;
+      }
+    }
+  }
+
+  label {
+    font-size: 13px;
+  }
+
   .form-wrapper {
-    margin: auto;
-    max-width: 350px;
-    padding: 15px 0;
+    width: 300px;
```

Comment:
What's this change from max to a smaller fixed width? Are we not expecting some stretching anymore?

Reply:
Stretching is now done by the form-wrapper as I [describe here](https://github.com/mozilla/activity-stream/pull/3965#discussion_r168813353)